### DISCUSSION TITLE
How do I set book meta information for my files?

### DISCUSSION THREAD

**Discussion by Murph101609** [TortugaPower/BookPlayer#878] [Q&A]
iPhone 11 Pro max
Newest version (BookPlayer)

how do I add an author when it says unknown?  Also can I add a book image if it doesn’t have one? 

 - OS Version: [e.g. 13.5.1]
 - App Version: [e.g. 3.9.0-1]

## Steps to reproduce the behavior
1. 
2. 
3.

**Comment by pichfl** [comment]
Your audio files are missing IDE3 tags. One free tool to change these would be [kid3](https://kid3.kde.org).

**Comment by dzaffaires** [comment]
Hi,
I've been importing files in BookPlayer with seemingly correct IDE3 tags, specifically the Author tag, and I keep getting "Unknown Author" written under the title of the file itself (in the app and in CarPlay).

I've used kid3 to check the files, and the Author tag seems OK. Is there a way to validate what's not valid for BookPlayer? 

<img width="402" height="155" alt="IDE3-Author" src="https://github.com/user-attachments/assets/09cce678-d70b-42f2-a4cb-82ac27266ebc" />

EDIT : So I figured it out with more trial and error. The "Author" tag used by BookPlayer is in fact the "Artist" tag. By setting a value in this field with kid3, BookPlayer recognizes it and shows the information without having to enter it manually.
